modules/checking-mco-status.adoc

```diff
@@ -14,35 +14,53 @@ To see the status of the Machine Config Operator, its sub-components, and the re
 [source,terminal]
```

Comment:
I am unable to comment on the introductory line 9 directly, so I'll add a suggestion here. You could add a reference to the "MCO" acronym after the first use of the name "Machine Config Operator":

> Machine Config Operator (MCO)